Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM php:8.1-cli
+
+RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
+
+# install dependencies
+RUN apt update && apt install -y --no-install-recommends \
+        libvips42 \
```

Comment:
Let's hold off the dependencies for vips until we are able to actually support it.